CorsixTH/Lua/game_ui.lua

```diff
@@ -1144,6 +1126,10 @@ function GameUI:afterLoad(old, new)
   if old < 122 then
     self.multigesturemove = {x = 0.0, y = 0.0}
   end
+  if old < 128 then
```

Comment:
Check if 128 is the right value...?

Reply:
We'll check this at the time it's merged - depends what else goes in between now and then.

Reply:
Alright, thanks.